src/webhint-theme/scripts/custom-renderer.js

```diff
@@ -48,7 +48,7 @@ renderer.code = (code, lang) => {
 };
 
 renderer.link = (href, title, text) => {
-    return `<a${isExternalLink(href) ? ' target="_blank"' : ''} href="${href}" title="${title}">${text}</a>`;
+    return `<a${isExternalLink(href) ? ' target="_blank"' : ''} href="${href}" ${title ? ` title="${title}"` : ''}>${text}</a>`;
```

Comment:
One question to @webhintio/contributors and @jaspreet57, what if instead of `''` we add the `text` as title if title doesn't exists?

Reply:
Yeah I thought of doing the same. 
But what if text is very long one? Sometime developer may make a paragraph as a link.
Or developer may not want to have a title for some links.

So leaving title as empty by default gives the developer a choice to decided it himself.